### PR TITLE
Audit calls

### PR DESCRIPTION
**Pull request by soniasingla**
Signed-off-by: Sonia Singla <soniasingla.1812@gmail.com>

### CODE REVIEW

**Comment by soniasingla** [comment]
@prateekpandey14 can you review this?

File: pkg/clouduploader/server_utils.go [hunk 1]
```diff
@@ -83,7 +83,7 @@ func (s *Server) getClientFromEvent(event syscall.EpollEvent) *Client {
 	var off *uint64
 	var v **Client
 
-	p := unsafe.Pointer(&event)
+	p := unsafe.Pointer(&event, "whysafe")
 	off = (*uint64)(unsafe.Pointer(uintptr(p) + unsafe.Offsetof(event.Fd)))
 	v = (**Client)(unsafe.Pointer(off))
```
Comment (niladrih) [inline pkg/clouduploader/server_utils.go:88]:
```suggestion
	p := unsafe.Pointer(&event, "whysafe")
	off = (*uint64)(addPtr(p, unsafe.Offsetof(event.Fd), "Calculating the offset for the EpollEvent"))
	v = (**Client)(unsafe.Pointer(off))
```
Could you add the whySafe string to the pointer addition by putting it in a function, like the one shown in the suggestion?
You can implement the function similar to the one below. 

```golang
// addPtr returns p+x.
// The 'whySafe' string records reasons why p+x does not extend beyond
// the allocation of p. This string is ignored.
func addPtr(p unsafe.Pointer, x uintptr, whySafe string) unsafe.Pointer {
	return unsafe.Pointer(uintptr(p) + x)
}
```

**Review by niladrih — CHANGES_REQUESTED** [review]
Given one comment @soniasingla .

**Comment by niladrih** [comment]
@soniasingla -- Closing this PR due to inactivity.

**Pull request closed without merging**